debug.php

```diff
@@ -785,9 +785,9 @@ function debug_handler_assertion_default_dg()
 		return function ($script, $line, $message)
 		{
 			$e = new Exception("{$script}:{$line} Assertion failed. {$message}");
-			if( getCurrentEnv() === 'production' )
+			if(\IFR\Main\App::isProd())
```

Comment:
Please do not use namespaced object inside functional interface.
If you wish to unify getCurrentEnv() values which is a good thing, please define const ENV_PRODUCTION etc. in functional way ( simple global const ) instead depending on object.

Generally, objects use functions, functions should never use objects.